client/me/purchases/purchase-item/index.tsx

```diff
@@ -626,24 +629,40 @@ export function PurchaseItemPaymentMethod( {
 		);
 	}
 
+	const goToAddPaymentMethod = ( e: React.MouseEvent< HTMLDivElement >, siteId: number ) => {
+		e.preventDefault();
+		e.stopPropagation();
```

Comment:
You probably don't need this as the Button component should already handle that.

Reply:
I need this because otherwise the parent anchor redirects to a details page.